### ISSUE TITLE
VisualStudioGallery and uploading vsix - Error Log Reference

### ISSUE THREAD

**Issue by 3F** [3F/vsCommandEvent#1]
How many bugs I should to see again on this week from MS -_-

again ! fully unclear the error message:

```
If you see this page, it is because an error occurred in the system while trying to process your request. We apologize for the inconvenience. This error has been reported to our team for analysis.

Error Log Reference #701f3f0a-b235-4599-bee7-23e86569bd93

Return to the site homepage. 
```

and... what I should to do o_O ?

My question **here**: https://social.msdn.microsoft.com/Forums/vstudio/en-US/5a8a8c9d-b99c-4add-8b45-f176325e115c/
_(I don't know where bug tracker for visualstudiogallery.msdn.microsoft.com)_

**But important note:** Do not use the [upload] type anymore (many problems and inconvenience...)
_[vsSBE project](https://visualstudiogallery.msdn.microsoft.com/0d1dbfd7-ed8a-40af-ae39-281bfeca2334) used the [link] type and feels good_

![vsgallery](https://cloud.githubusercontent.com/assets/3246589/19536856/766e8d40-9656-11e6-98b8-642abd1d293b.png)

**Comment by 3F** [comment]
```csharp
The extension targets Visual Studio "15" but is missing a setup file '/manifest.json'.
v
The extension targets Visual Studio "15" but is missing a setup file '/catalog.json'.
v
This extension targets Visual Studio "15" but is missing setup dependencies.
```

If try to use VS 15 and: 

```xml
<GenerateVsixV3>true</GenerateVsixV3>
```

even for new empty VSIX project I only see manifest.json, for example:

```json
{"id":"VSIXProject1.Company.e29eb6bb-8dc4-48a4-8cd4-db59fc7d4d41","version":"1.0","type":"Vsix","language":"en-US","files":[{"fileName":"/extension.vsixmanifest","sha256":"01E9CA3C115CE69D3ED86EC701405280CAA562DDE64D0DB039056DC912706AC1"}]}
```
/catalog.json + some dependencies, still as a error for upload in VS Gallery even for new empty project (template: `Extensibility` - `VSIX Project`)

**Related this:**
* https://github.com/3F/vsSolutionBuildEvent/issues/36

> Tim Sneath 
> We haven't yet published documentation on the new extension format, but plan to do so within the next month. 

time will tell...

**Comment by 3F** [comment]
For https://visualstudiogallery.msdn.microsoft.com/ad9f19b2-04c0-46fe-9637-9a52ce4ca661

I released other build v1.2.0.**12141** 

The only difference between v1.2.0.**11076** and v1.2.0.**12141**:

vsix package in 12141 does not contain the definition - `VisualStudio Version="15.0"` in source.extension.vsixmanifest

**Comment by 3F** [comment]
MS changed the distribution type for this plugin manually.
Modern packages will be distributed in the same way like https://github.com/3F/vsSolutionBuildEvent

ie. SDK10 & SDK15

* Today's VS Marketplace for vsCommandEvent: https://marketplace.visualstudio.com/items?itemName=GitHub3F.vsCommandEvent

**VS2019** support is considering now via #2. Closed.
